RocketChat/Rocket.Chat
# [IMPROVE][ENTERPRISE] Allow mapping LDAP groups to multiple RC roles

## Proposed changes (including videos or screenshots)


- Add support to mapping LDAP groups to multiple roles (by specifying arrays in the "User Data Group Map" enterprise setting.



## Issue(s)

[Task - ClickUp](https://app.clickup.com/t/1qmnxc2)

## Steps to test or reproduce

Go to **Administration > LDAP > Enterprise > User Data Group Map** and map a LDAP group to multiple roles using an array (e.g. `"manager":["leader", "moderator"]`). Login as a LDAP user who is in the specified group (e.g. `manager`).
**_Expected results:_** the user should now be given all the roles specified in the array.

## Further comments

Mapping a LDAP group to a single role (as a string, e.g. `"rocket-admin": "admin"`) is still supported.
Waiting on #23824
